src/js/common/schemaform/review/ArrayField.jsx

```diff
@@ -150,19 +150,26 @@ class ArrayField extends React.Component {
       pageKey: fieldName
     };
 
+    // TODO: Make this better; it's super hacky for now.
```

Comment:
I think the non-hacky way to fix this is to get rid of our special casing of array data. But that needs an ok from Alex (there's an issue somewhere).